src/panels/Menu/hooks.ts

```diff
@@ -41,7 +44,7 @@ export function useMenuItems() {
   }, [isSkyBrowserEnabled, enablePanel]);
 
   function setMenuItemVisible(id: string, value: boolean) {
```

Comment:
See comment thread above, but I suggest that we remove this function entirely and just use the `dispatch` in the code

Then there is no need for the renaming of the function